foundation-extensions/mobile-core/identity/identity-api-reference.md

```diff
@@ -810,7 +667,7 @@ ACPIdentity.getExperienceCloudId(success, fail);
 * _success_ is a callback containing the experience cloud id if the `getExperienceCloudId` API executed without any errors.
```

Comment:
There are a few places where ECID is referenced as "experience cloud id". Should this be capitalized instead to read "Experience Cloud ID"?  Or Simply "ECID"?

Reply:
It should DEFINITELY be Experience Cloud ID if we're referring it to it by the full name.

Reply:
Thanks @calebk1 . I've changed the references to "ECID" as the start of the section already establishes "Adobe Experience Cloud ID (ECID)".

Reply:
Yup! That's 100% okay - since we've already referred to ECID as "Experience Cloud ID" earlier, we can just shorten it to ECID in subsequent usages.